### PR TITLE
Fixing spacing around payment buttons

### PR DESCRIPTION
**Pull request by vojtechmzl**
Hey @ScottDowne, here is a fix for the spacing issue. Thanks for flagging it.

I didn't know how else to select the elements, so I added new IDs. Let me know if there is a better way.

Thanks.

### CODE REVIEW

File: less/pages/single-form.less [hunk 1]
```diff
@@ -1,3 +1,11 @@
 .payment-section {
   padding-top: 25px;
 }
+
+#section-heading {
+  margin-bottom:14px;
+}
+
+#privacy-policy p {
+  margin-bottom:0px;
```
Comment (ScottDowne) [inline less/pages/single-form.less]: I was wrong, I don't think this is needed. Thoughts?

File: less/pages/single-form.less [hunk 2]
```diff
@@ -1,3 +1,11 @@
 .payment-section {
   padding-top: 25px;
 }
+
+#section-heading {
+  margin-bottom:14px;
```
Comment (ScottDowne) [inline less/pages/single-form.less]: I think we can accomplish the same thing by removing this line: https://github.com/mozilla/donate.mozilla.org/blob/971cb3793f90b94c9a3b73d8b95f8dfee7e8938d/less/shared.less#L472

Comment (ScottDowne) [inline less/pages/single-form.less]: Roughly the same thing.

Comment (vojtechmzl) [inline less/pages/single-form.less]: Great find. Just pushed an update with the simplified code @ScottDowne. Thanks!

File: less/pages/single-form.less (hunk 1 above)
Comment (vojtechmzl) [inline less/pages/single-form.less]: Agreed. Removed from the latest push.

**Pull request merged**